enterprise/server/scheduling/task_leaser/task_leaser.go

```diff
@@ -67,7 +67,9 @@ func (t *TaskLeaser) pingServer(ctx context.Context) (b []byte, err error) {
 		ExecutorId:        t.executorID,
 		TaskId:            t.taskID,
 		SupportsReconnect: *enableReconnect,
-		ReconnectToken:    t.reconnectToken,
```

Comment:
hm was deleting this line intentional? Seems like if `supportsReconnect` is false then we should keep setting it the old way? or if we don't care about rollouts, should we just instead unconditionally set this to lease ID and remove the `supportsReconnect` field?

Reply:
`supportsReconnect` is true if either the new `supports_reconnect` field is set OR the old `reconnect_token` field is set which means the ReconnectToken should be populated in all cases where it was previously populated.